samples/application/simple-go/main.go

```diff
@@ -200,7 +200,19 @@ func main() {
 
 	// Invoke FPC Chaincode
 	logger.Infof("--> Invoke FPC chaincode: %s", contract.Name())
-	result, err := contract.SubmitTransaction("myFunction", "arg1", "arg2", "arg3")
+	var funcion, arg1, arg2, arg3 string
+	if contract.Name() == "echo" {
+		funcion = "myFunction"
+		arg1 = "arg1"
+		arg2 = "arg2"
+		arg3 = "arg3"
+	} else {
+		funcion = "storeAsset"
+		arg1 = "asset1"
+		arg2 = "200"
+		arg3 = ""
+	}
+	result, err := contract.SubmitTransaction(funcion, arg1, arg2, arg3)
```

Comment:
This needs refactoring (even as it currently is).
The current version refers to the "echo" chaincode, but it's not written anywhere (I think).
The new version, additionally, makes this "client" work even with the helloworld.
Ideally, we should separate the two, clearly indicating the echo client and the helloworld client.

Reply:
This will be covered in #605